pytorch_toolkit/object_detection/custom-detection/readme.md

```diff
@@ -0,0 +1,135 @@
+# Custom object detector
+
+Custom object detectors are lightweight object detection models that have been pre-trained on MS COCO object detection dataset.
+It is assumed that one will use these pre-trained models as starting points in order to train specific object detection models (e.g. 'cat' and 'dog' detection).
+There was no a goal to train lightweight ready-to-use 80 class (MS COCO classes) detector.
+
+| Model Name | Complexity (GFLOPs) | Size (Mp) | AP @ [IoU=0.50:0.95] (%) | Links | GPU_NUM |
+| --- | --- | --- | --- | --- | --- |
+| mobilenet_v2-2s_ssd-256x256 | 0.86 | 1.99 | 11.3 | [snapshot](https://download.01.org/opencv/openvino_training_extensions/models/object_detection/v2/mobilenet_v2-2s_ssd-256x256.pth), [configuration file](./mobilenet_v2-2s_ssd-256x256/config.py) | 3 |
+| mobilenet_v2-2s_ssd-384x384 | 1.92 | 1.99 | 13.3 | [snapshot](https://download.01.org/opencv/openvino_training_extensions/models/object_detection/v2/mobilenet_v2-2s_ssd-384x384.pth), [configuration file](./mobilenet_v2-2s_ssd-384x384/config.py) | 3 |
+| mobilenet_v2-2s_ssd-512x512 | 3.42 | 1.99 | 12.7 | [snapshot](https://download.01.org/opencv/openvino_training_extensions/models/object_detection/v2/mobilenet_v2-2s_ssd-512x512.pth), [configuration file](./mobilenet_v2-2s_ssd-512x512/config.py) | 3 |
+
+## Training pipeline
+
+### 0. Change a directory in your terminal to object_detection.
+
+```bash
+cd <openvino_training_extensions>/pytorch_toolkit/object_detection
+```
+
+### 1. Select a training configuration file and get pre-trained snapshot if available. Please see the table above.
+
+```bash
+export MODEL_NAME=mobilenet_v2-2s_ssd-256x256
+export CONFIGURATION_FILE=./custom-detection/$MODEL_NAME/config.py
+```
+
+### 2. Collect dataset
+
+You can train a model on existing toy dataset `openvino_training_extensions/data/airport`. Obviously such dataset is not sufficient for training good enough model.
+
+### 3. Prepare annotation
+
+The existing toy dataset has annotation in the Common Objects in Context (COCO) and mmdetection CustomDataset format.
```

Comment:
There was CustomCocoDataset that was introduced by me, but I had to remove it since the same feature (custom coco classes) appeared in CocoDataset. CustomDataset is not COCO-like format ([see here ](https://github.com/opencv/mmdetection/blob/feature/ik/more_tools_with_update_config/mmdet/datasets/custom.py#L13))